etc/development/config-web.toml

```diff
@@ -1,9 +1,10 @@
 [webserver]
-# host = "localhost"
-# port = 16677
+host = "0.0.0.0"
+port = 16677
 
 [amqp]
 uri = "amqp://guest:guest@localhost:5672"
 
 [database]
-uri = "sqlite:///secpi-database-development.sqlite"
+#uri = "sqlite:///secpi-database-development.sqlite"
+uri = "mysql+pymysql://secpi:secret@localhost/secpi"
```

Comment:
Don't cheat ;]. And apologies again for my pickyness.
```suggestion
uri = "mysql+pymysql://secpi:secret@localhost/secpi-development"
```

Reply:
;-)

Reply:
Fixed with 5237e3aaf.